core/server/src/main/java/alluxio/master/AlluxioMaster.java

```diff
@@ -71,12 +71,13 @@
    */
   public static void main(String[] args) {
     if (args.length != 0) {
-      LOG.info("java -cp {} {}", Version.ALLUXIO_JAR, AlluxioMaster.class.getCanonicalName());
+      LOG.info("java -cp {} {}", RuntimeConstants.ALLUXIO_JAR,
+          AlluxioMaster.class.getCanonicalName());
       System.exit(-1);
     }
 
     // validate the conf
-    if (!ValidateConf.validate()) {
+    if (!ValidateConf.validate(Configuration.createServerConf())) {
```

Comment:
Yes, it could cause clients to exit unexpectedly, but that might be the correct behavior if the configuration is flawed? Otherwise the client behavior might be unintended.